MongoDB/recipes/default.rb

```diff
@@ -0,0 +1,37 @@
+#
+# Cookbook:: MongoDB
+# Recipe:: default
+#
+# Copyright:: 2017, The Authors, All Rights Reserved.
+
+#'MongoDB::importpkey'
+apt_repository 'import_key' do
+  uri 'hkp://keyserver.ubuntu.com:80'
+  components ['main']
+  distribution 'trusty'
+  key '7F0CEB10'
+  keyserver 'keyserver.ubuntu.com'
+  action :add
+  deb_src true
+end
+
+#'MongoDB::createfile'
+apt_repository 'create_file' do
+uri 'hkp://keyserver.ubuntu.com:80'
+  components ['main']
+  distribution 'trusty'
+cookbook 'sudo tee /etc/apt/sources.list.d/mongodb-org-3.0.list'
+end
+
+#'MongoDB::update'
+apt_update 'update' do
```

Comment:
I guess we are missing something here. 
apt update is not required when we use "apt_repository" resource. Please run this on a new AWS instance.